python/utils/spacy_eval.ipynb

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
this notebook should be in a notebooks folder in the root directory (along with the other notebooks in the exploratory_nbs folder); typically the python folder should only contain .py files.